js/ui/components/clubhouse.js

```diff
@@ -114,12 +115,33 @@ var ClubhouseNotificationSource = new Lang.Class({
     },
 });
 
+var NotifiableButton = new Lang.Class({
+    Name: 'NotifiableButton',
+    Extends: St.Button,
+
+    _init: function(params) {
+        this.parent(params);
+
+        this.connect('clicked', () => {
+            this._stopPulsing();
+        })
+    },
+
+    pulsate: function() {
+        this.get_children()[0].add_style_pseudo_class('highlighted');
```

Comment:
We should be able to call this on the icon directly, instead of getting the first child which is not set by the button directly. So instead of allowing any child to be passed to this button, we should stop passing the icon and instead create and add it here (in the constructor), since this isto become the "open button" only and not a generic notifiable button.